content/en/docs/tasks/traffic-management/egress/egress-gateway/index.md

```diff
@@ -543,7 +543,7 @@ external service.
     sleep istio-proxy
     {{< /text >}}
 
-1.  Create the same destination rule as for the `sleep` pod in the `default` namespace to direct the traffic through the egress gateway:
+1.  Create the same destination rule as for the `sleep` pod above, just now in the `test-egress` namespace, to direct the traffic through the egress gateway:
```

Comment:
```suggestion
1.  Create a similar destination rule as used for the `sleep` pod in the `default` namespace, to direct the `test-egress` namespace traffic through the egress gateway:
```

Reply:
I like this text and have committed it to the PR :+1: